ccd-definition/CaseEventToFields/CareSupervision/messageJudge.json

```diff
@@ -47,7 +47,8 @@
     "ShowSummaryChangeOption": "N",
     "PageDisplayOrder": 2,
     "FieldShowCondition": "isMessageRegardingC2=\"DO_NOT_SHOW\"",
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/message-judge/mid-event"
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/message-judge/mid-event",
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/message-judge/validate-email/mid-event"
```

Comment:
yes because message-judge/mid-event is called on trigger of radio button selection 'reply' to message or 'respond' to message, so this mid event prepopulates the next page with some fields like default email.. the next mid event (one o i have added) then grabs what email has been added in this page, not the default one.

Reply:
generated spreadsheet contains only one mid event, so one is lost i think

Reply:
changed :)